lib/routes.js

```diff
@@ -35,8 +36,9 @@ const statsClient = new StatsClient(localCacheClient, STATS_INTERVAL,
     STATS_EXPIRY);
 
 function checkUnsuportedRoutes(req, res, log) {
-    if (req.query.policy !== undefined ||
-        req.query.tagging !== undefined) {
+    const unsupportedRequestQuery = Object.keys(req.query)
```

Comment:
Yes, that's the duplication I was mentioning about. You can parse the url once in there and use it everywhere.